main.py

```diff
@@ -14,10 +15,12 @@
 
 api_base = 'https://api.github.com'
 
-auth_url = 'https://github.com/login/oauth/authorize'\
-    '?client_id={}'.format(client_id)
+auth_url = 'https://github.com/login/oauth/authorize' \
+           '?client_id={}'.format(client_id)
 
-time_range = '2019-10-01T00:00:01Z..2019-10-31T23:59:59'
+year = str(datetime.datetime.now().year)
+
+time_range = year + '-10-01T00:00:01Z..' + year + '-10-31T23:59:59'
```

Comment:
please use string formatting to add the year to this string, so you don't need an additional var
```
time_range = '{year}...{year}...'.format(year=datetime....)
```